sde_collections/tasks.py

```diff
@@ -141,3 +143,59 @@ def resolve_title_pattern(title_pattern_id):
     TitlePattern = apps.get_model("sde_collections", "TitlePattern")
     title_pattern = TitlePattern.objects.get(id=title_pattern_id)
     title_pattern.apply()
+
+
+@celery_app.task
+def fetch_and_update_full_text(collection_id, server_type):
```

Comment:
is this server type or server name?

Reply:
Changed it to server_name for better clarity.